### ISSUE TITLE
Training Teacher Model

### ISSUE THREAD

**Issue by MovingKyu** [nlpyang/NoisySumm#3]
First, I would like to thank you for publicizing the code.

I am wondering how to define "MODEL_PATH" in training the teacher model. Your current code seems to use a pre-trained one. Could you specify which pre-trained model you have used? 

Second, why is the "SPLIT" and "INPUT_JSON" set to test set of the data. This seems a little weird for me. Do you have any explanation why? 

Thank you!
